example-widget-semantic-ui/src/DicePage/DicePage.test.tsx

```diff
@@ -60,22 +60,26 @@ describe('<DicePage />', () => {
   it('should request the capabilities', async () => {
     render(<DicePage />, { wrapper });
 
-    expect(widgetApi.requestCapabilities).toBeCalledWith([
-      WidgetEventCapability.forRoomEvent(
-        EventDirection.Receive,
-        'net.nordeck.throw_dice',
-      ),
-    ]);
+    await waitFor(() => {
```

Comment:
Test still runs event if I revert all changes here.
Can you check this as well and eventually revert them?

Reply:
pretty sure this was failing at some point, otherwise wouldn't have it there. Anyhooo, seems ok now.